.github/workflows/release.yaml

```diff
@@ -197,8 +199,8 @@ jobs:
             docker manifest push $image_name
             docker manifest push quay.io/$image_name
 
-            cosign sign -y --key env://COSIGN_PRIVATE_KEY quay.io/$image_name
-
+            digest=$(docker inspect quay.io/$image_name | jq -r '.[0].RepoDigests[0]' | cut -d'@' -f2)
```

Comment:
https://cloud.google.com/kubernetes-engine/docs/concepts/about-container-images#docker_client

Reply:
Maybe we need to use `docker manifest inspect` as the image isn't built in the same job.
I'll look into.

Reply:
I'm still testing, but digests are different by platform (linux/amd64, linux/arm64, windows/amd64), so we need to sign images by platform.
And we need to use `docker manifest inspect`.

Reply:
Fixed. 720fdfc2e46f554d149412e9dbea43d2a93de25e